File: 2107-find-unique-binary-string/find-unique-binary-string.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        nums.sort()
        n = len(nums[0])
        for i in range(len(nums)):
            x = format(i, f'0{n}b')
            if x != nums[i]:
                return x
        return format(len(nums), f'0{n}b')

File: 2107-find-unique-binary-string/find-unique-binary-string.js
var findDifferentBinaryString = function(nums) {
    nums.sort();
    const n = nums[0].length;
    for (let i = 0; i <= nums.length; i++) {
        const x = i.toString(2).padStart(n, '0');
        if (i === nums.length || x !== nums[i]) return x;
    }
};